tests/e2e/statefulsets.go

```diff
@@ -93,6 +100,32 @@ var _ = ginkgo.Describe("statefulset", func() {
 		if err == nil && service != nil {
 			deleteService(namespace, client, service)
 		}
+		if stretchedSVC {
+			zonalPolicy = GetAndExpectStringEnvVar(envZonalStoragePolicyName)
+			labels_ns = map[string]string{}
+			labels_ns[admissionapi.EnforceLevelLabel] = string(admissionapi.LevelPrivileged)
+			labels_ns["e2e-framework"] = f.BaseName
+
+			if zonalPolicy == "" {
+				ginkgo.Fail(envZonalStoragePolicyName + " env variable not set")
+			}
+			zonalWffcPolicy = GetAndExpectStringEnvVar(envZonalWffcStoragePolicyName)
+			if zonalWffcPolicy == "" {
```

Comment:
This if block is not required, it is already handled in GetAndExpectStringEnvVar